go.sum

```diff
@@ -36,6 +36,8 @@ dmitri.shuralyov.com/gpu/mtl v0.0.0-20190408044501-666a987793e9/go.mod h1:H6x//7
 github.com/BurntSushi/toml v0.3.1 h1:WXkYYl6Yr3qBf1K79EBnL4mak0OimBfB0XUf9Vl28OQ=
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
 github.com/BurntSushi/xgb v0.0.0-20160522181843-27f122750802/go.mod h1:IVnqGOEym/WlBOVXweHU+Q+/VP0lqqI8lqeDx9IjBqo=
+github.com/Didainius/go-vcloud-director/v2 v2.10.1-0.20210205074351-69c21764466c h1:LDfZUIeCOVpZgMf+mPIgydkfscRwBPWVmeANWlEd/Hc=
+github.com/Didainius/go-vcloud-director/v2 v2.10.1-0.20210205074351-69c21764466c/go.mod h1:czvTQZlB4/WsOsL7rMVCb+SYAPJhx/dYoS/Sk7rc/O0=
```

Comment:
Just in case, it's pointing to Didainius/

Reply:
`go mod tidy` cleaned it up, but no biggie - these only hold checksum, not necessarily actual imports.